main.py

```diff
@@ -1,7 +1,88 @@
 import init_django_orm  # noqa: F401
 
+from db.models import Genre, Actor
 from django.db.models import QuerySet
 
+list_of_genres = [
+    "Western", "Action", "Dramma"
+]
+
+list_of_actors = [
+    {
+        "first_name": "George",
+        "last_name": "Klooney"
+    },
+    {
+        "first_name": "Kianu",
+        "last_name": "Reaves"
+    },
+    {
+        "first_name": "Scarlett",
+        "last_name": "Keegan"
+    },
+    {
+        "first_name": "Will",
+        "last_name": "Smith"
+    },
+    {
+        "first_name": "Jaden",
+        "last_name": "Smith"
+    },
+    {
+        "first_name": "Scarlett",
+        "last_name": "Johansson"
+    },
+
+]
```

Comment:
You could probably use lists of tuples for the source of actor, that will allow you to write somewhat less verbose code in this case, although if you expect to use another types of sources in the future, like json objects for example, this approach could be better